db.go

```diff
@@ -273,9 +272,9 @@ func Open(opt Options) (db *DB, err error) {
 		}
 	}()
 
-	elog := y.NoEventLog
-	if opt.EventLogging {
-		elog = trace.NewEventLog("Badger", "DB")
+	logger := NilLogger
+	if opt.Logging {
+		logger = defaultLogger
```

Comment:
You can remove the 3 lines above. The `y.NoEventLog` was added because `eventLog` would take up too much of CPU/memory. Now we don't need it.